bc_obps/registration/api/v2/_operations/operation_id.py

```diff
@@ -29,3 +30,33 @@
 @handle_http_errors()
 def get_operation_v2(request: HttpRequest, operation_id: UUID) -> Tuple[Literal[200], Operation]:
     return 200, OperationService.get_if_authorized(get_current_user_guid(request), operation_id)
+
+
+@router.get(
+    "/v2/operations/{uuid:operation_id}/with-documents",
+    response={200: OperationOutWithDocuments, codes_4xx: Message},
+    tags=OPERATION_TAGS,
+    description="""Retrieves the details of a specific operation by its ID along with it's documents""",
+    exclude_none=True,
+    auth=authorize("approved_authorized_roles"),
+)
+@handle_http_errors()
+def get_operation_with_documents(request: HttpRequest, operation_id: UUID) -> Tuple[Literal[200], Operation]:
+    return 200, OperationService.get_if_authorized(get_current_user_guid(request), operation_id)
```

Comment:
Added a new API route, though I was split on if I should add a query param like `documents` and return the documents if it was true. Can we have conditional Django Ninja schemas? Is a separate route fine?